scripts/microbe_stage/setup.as

```diff
@@ -239,6 +240,11 @@ void onReturnFromEditor(CellStageWorld@ world)
     Species::copyProcessesFromSpecies(world, ourActualSpecies, player);
 
     PlayerSpeciesSpawner factory("Default");
+
+    // Offset between cells
+    pos._Position.X += membraneComponent.calculateEncompassingCircleRadius();
+    pos._Position.Z += membraneComponent.calculateEncompassingCircleRadius();
+
```

Comment:
There should be `pos.Marked = true;` here. As anytime a component is touched it needs to be marked. Otherwise the systems don't pick up the change.